#camera_node to path_node
float64 a
float64 b
float64 state
#bool state
